OracleManagementAgent/dockerfiles/latest/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please remove "All rights reserved." as we no longer include this phrase.

Reply:
fixed